cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSIMongoSinkTest.java

```diff
@@ -64,9 +67,9 @@ public NGSIMongoSinkTest() {
     private NotifyContextRequest.ContextElement createContextElement() {
         NotifyContextRequest notifyContextRequest = new NotifyContextRequest();
         NotifyContextRequest.ContextMetadata contextMetadata = new NotifyContextRequest.ContextMetadata();
-        contextMetadata.setName("location");
-        contextMetadata.setType("string");
-        contextMetadata.setContextMetadata(new JsonPrimitive("WGS84"));
+        contextMetadata.setName("TimeInstant");
+        contextMetadata.setType("recvTime");
```

Comment:
NTC. I think metadata type is ignored but it would be better to set "DateTime" instead of "recvTime"